backstop/tests/tabs.html

```diff
@@ -698,212 +706,237 @@ <h2>Borderless</h2>
+              </svg>
+              <span class="iui-tab-label">
+                <div>Left</div>
+                <div class="iui-tab-description">Sublabel</div>
+              </span>
+            </button>
+          </li>
+          <li>
+            <button
+              class="iui-tab"
+              role="tab"
+              aria-selected="false"
+              aria-controls="disabled-tab"
+              disabled
+            >
+              <!-- Use placeholder.svg from @itwin/itwinui-icons package. -->
+              <svg
+                class="iui-tab-icon"
+                aria-hidden="true"
+                viewBox="0 0 16 16"
+              >
+                <path
+                  d="m0 0v16h16v-16zm14.85714 1.94286v12.11428l-6.05714-6.05714zm-6.85714 5.25714-6.05714-6.05714h12.11428zm-.8.8-6.05714 6.05714v-12.11428zm.8.8 6.05714 6.05714h-12.11428z"
+                />
+              </svg>
+              <span class="iui-tab-label">
+                <div>Disabled</div>
+                <div class="iui-tab-description">Sublabel</div>
+              </span>
+            </button>
+          </li>
+          <li>
+            <button
+              class="iui-tab"
+              role="tab"
+              aria-selected="false"
+              aria-controls="none-tab"
+            >
+              <span class="iui-tab-label">None</span>
+            </button>
+          </li>
+        </ul>
+        <div class="iui-tab-stripe"></div>
+        <div
+          class="iui-tabs-content"
+          role="tabpanel"
+          id="active-tab"
+          aria-labelledby="active"
+        >
+          Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod
+          tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim
+          veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea
+          commodo consequat. Duis aute irure dolor in reprehenderit in voluptate
+          velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat
+          cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id
+          est laborum.
+        </div>
       </div>
 
     </section>
+
+    <script>
+      document.querySelectorAll('.iui-tabs:not(.iui-default)').forEach((tabs) => {
+        const wrapper = tabs.parentElement;
+        const activeTab = tabs.querySelector('.iui-active');
+        const stripe = wrapper.querySelector('.iui-tab-stripe');
+        const setStripe = (activeEl) => {
+          stripe.style.left = `${activeEl.offsetLeft}px`;
+          stripe.style.width = `${activeEl.offsetWidth}px`;
+        }
+        setStripe(activeTab);
+
+        [...tabs.children].forEach(({ firstElementChild: tab }) => {
+          tab.addEventListener('click', () => {
+            tabs.querySelector('.iui-active')?.classList.remove('iui-active');
+            tab.classList.add('iui-active');
+            setStripe(tab); // needs to be set in react for animation
```

Comment:
@veekeys @bentleyvk Need opinion about this. These inline styles may not be "nice" but the resulting animation is.

The alternative is to use css variables for `left` and `width`, that we can update in react.

Reply:
That animation look nice! I don't think there should be too much code to handle this in React.